### PR TITLE
pr-hitl-integration requested changes 

### PR DESCRIPTION
**Pull request by arwagoner**

### CODE REVIEW

File: src/gazebo_mavlink_interface.cpp [hunk 1]
```diff
@@ -412,8 +411,8 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
 
   if (_sdf->HasElement("imu_rate")) {
     imu_rate_ = _sdf->GetElement("imu_rate")->Get<int>();
+    imu_rate_ = 1/imu_rate_;
```
Comment (TSC21) [inline src/gazebo_mavlink_interface.cpp:414]: @arwagoner where's the `set_imu_rate` variable which was set when the sdf element is detected?

File: src/gazebo_mavlink_interface.cpp [hunk 2]
```diff
@@ -712,6 +710,8 @@ void GazeboMavlinkInterface::ImuCallback(ImuPtr& imu_message) {
       imu_message->angular_velocity().z()));
     math::Vector3 mag_b = q_nb.RotateVectorReverse(mag_n) + mag_noise_b;
 
+  if (imu_rate_!=0 && dt >= imu_rate_)
```
Comment (TSC21) [inline src/gazebo_mavlink_interface.cpp:713]: Ok understood what you did.

File: src/gazebo_mavlink_interface.cpp (hunk 1 above)
Comment (arwagoner) [reply src/gazebo_mavlink_interface.cpp:414]: I deleted it because it seemed unnecessary.

**Review by TSC21 — APPROVED** [review]
LGTM. Did you get the chance to retest?

**Comment by arwagoner** [comment]
Yes, I did retest. Everything works as expected.

File: include/msgbuffer.h [hunk 3]
```diff
@@ -5,22 +5,13 @@
...
+ * Copyright 2014,2015,2016 Vladimir Ermakov, All rights reserved.
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
-*/
+ * This file is part of the mavros package and subject to the license terms
```
Comment (LorenzMeier) [inline include/msgbuffer.h:11]: @TSC21 This doesn't work. We need this file with MAVROS attribution (NOT flagged as being part of the package) and under Apache 2.0 - or replace it entirely.

Comment (TSC21) [reply include/msgbuffer.h:11]: @LorenzMeier what do you mean with "MAVROS attribution"?

**Pull request merged**